From 6cce525ecdba416ada9e9c5166fc95f8d8c26582 Mon Sep 17 00:00:00 2001
From: John Kacur <jkacur@redhat.com>
Date: Fri, 18 Dec 2015 15:07:27 +0100
Subject: [PATCH 1/2] flightrecorder: trace-cmd.config Add the -i option

Add the -i option so that trace-cmd will ignore listed options it
doesn't find. This works around differences that might exist due to
different architectures for example

Reported-by: Chunyu Hu <chuhu@redhat.com>
Signed-off-by: John Kacur <jkacur@redhat.com>
---
 trace-cmd.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/trace-cmd.config b/trace-cmd.config
index a55a979dc5ce..d767f852479e 100644
--- a/trace-cmd.config
+++ b/trace-cmd.config
@@ -1,4 +1,4 @@
 # ftrace based flight recorder configuration file.
 
 # trace-cmd options
-OPTS="-b 2048 -e block -e irq -e mce -e module -e power -e sched -e signal -e sys_enter -e sys_exit -e timer -e workqueue"
+OPTS="-b 2048 -i -e block -e irq -e mce -e module -e power -e sched -e signal -e sys_enter -e sys_exit -e timer -e workqueue"
-- 
2.4.3

